pkg/backingstore/backingstore.go

```diff
@@ -350,9 +350,17 @@ func createCommon(cmd *cobra.Command, args []string, storeType nbv1.StoreType, p
 	o = util.KubeObject(bundle.File_deploy_internal_secret_empty_yaml)
 	secret := o.(*corev1.Secret)
 	secret.Namespace = options.Namespace
-	secret.Name = fmt.Sprintf("backing-store-%s-%s", storeType, name)
-	secret.StringData = map[string]string{}
-	secret.Data = nil
+	if secretName != "" {
```

Comment:
I thought that Liran meant this `if` in his comment. 
@baum, I think it's more readable to have the non-negative condition in an `if-else` clause
e.g:

```suggestion
	if secretName == "" {
	
```

Reply:
> @liranmauda I think that in this case, it makes sense to check for a breaking condition.
I actually thought that you meant the outer if. see comment there

True, In the example I gave I saw (wrongly) that the else belongs to that if. 
Generally speaking, where there is no else (and only if) it makes sense to write the negative condition, if not avoidable.

> I thought that Liran meant this `if` in his comment. @baum, I think it's more readable to have the non-negative condition in an `if-else` clause e.g:

This is the correct condition

Reply:
@dannyzaken @romayalon improved the verbosity of the PR description.